pdc/apps/release/views.py

```diff
@@ -694,6 +694,10 @@ def create(self, request):
         if source_release.releasecomponent_set.count() == 0:
             return Response({'detail': 'there is no component in source release'},
                             status=status.HTTP_204_NO_CONTENT)
+        if not target_release.active:
+            return Response({'detail': 'can\'t clone components' +
+                                       'for target release %s with inactive' % target_release_id},
+                            status=status.HTTP_406_NOT_ACCEPTABLE)
```

Comment:
It seems HTTP_406_NOT_ACCEPTABLE is according to Accept Headers. https://en.wikipedia.org/wiki/List_of_HTTP_status_codes
I think HTTP_400_BAD_REQUEST should be ok.